detect_secrets/plugins/__init__.py

```diff
@@ -145,7 +141,7 @@ def _convert_sensitivity_values_to_class_tuple(sensitivity_values):
              This way, we can initialize the class with <plugin_class_name>(<value>)
 
     Example:
-        >>> [ ('HexHighEntropyString', 3), ('PrivateKeyDetector', true), ]
+        >>> [ ('HexHighEntropyString', {'hex_limit': 3}), ('PrivateKeyDetector', {'private_key_detector': true}), ]
```

Comment:
please shorten this line. =)

Maybe,

```
>>> [
...    ('HexHighEntropyString', {'hex_limit': 3}),
...    ('PrivateKeyDetector', {'private_key_detector': true}),
... ]
```

?

To make it easier, I will adjust E501 settings for the future.